build-support/bin/ci.py

```diff
@@ -154,6 +163,24 @@ def set_run_from_pex() -> None:
   # integration tests that shell out to `./pants`, so we set this env var for those cases.
   os.environ["RUN_PANTS_FROM_PEX"] = "1"
 
+
+@contextmanager
+def get_remote_execution_oauth_token_path() -> Iterator[str]:
+  command = (
+    ["./pants.pex", "--quiet", "run", "build-support/bin:get_rbe_token"]
+    if os.getenv("CI")
+    else ["gcloud", "auth", "application-default", "print-access-token"]
```

Comment:
This will only work is the application-default has the correct rbe role.

Reply:
Correct. The assumption is that users are logged in to an account with the appropriate permissions, which the [How to use RBE doc](https://docs.google.com/document/d/1Vzc-kwFfmoKZcJDnGlk_Xtnzw9A6YMdcaSFHdRJ_-Zg/edit#heading=h.klt5lezcknmu) instructs to do. 

The command will fail if not, due to `check=True` in `subprocess.run()`.